slapd-cli

```diff
@@ -227,6 +227,17 @@ else
 	message "info" "[INFO] Using built-in configuration - this may cause some problems"
 fi
 
+# Use systemd to manage the service if it's present
+_use_systemctl=0
```

Comment:
Maybe add comments to specify the signification of PPID, SYSTEMCTL_SKIP_REDIRECT, and INVOCATION_ID?

Reply:
done